### PR TITLE
Migrate Cloud Foundry into using Parallel Deploy stage

### PR DESCRIPTION
**Pull request by gregturn**

### CODE REVIEW

**Comment by gregturn** [comment]
Supercedes #661

**Comment by gregturn** [comment]
Related to https://github.com/spinnaker/clouddriver/pull/209

**Comment by gregturn** [comment]
@cfieber Please review.

**Comment by gregturn** [comment]
Based on mods to deck and clouddriver, trimmed back several edits made in this PR.

Please review @cfieber, @ajordens, or @duftler

File: orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cf/CloudFoundryServerGroupCreator.groovy [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2014 Google, Inc.
```
Comment (ajordens) [inline orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cf/CloudFoundryServerGroupCreator.groovy]: Minor but this Copyright header isn't quite right.

**Comment by ajordens** [comment]
LGTM, I see some logic transitioned over to clouddriver which makes sense.

**Comment by gregturn** [comment]
@ajordens Okay, I made that edit.

**Pull request merged**